_data/simple-icons.json

```diff
@@ -12152,6 +12152,11 @@
             "source": "https://www.plume.com",
             "guidelines": "https://www.plume.com/legal/trademarks"
         },
+        {
+            "title": "plunk",
+            "hex": "000000",
+            "source": "https://www.useplunk.com"
```

Comment:
I would prefer if we can use the GitHub asset at https://github.com/useplunk/docs/blob/main/logo/plunk-light.svg as the source. Hopefully you shouldn't have to make any changes as I assume they are the same file as on the website, but check first!